_board/0xcb_gemini.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: download
+board_id: "0xcb_gemini"
+title: "0xCB Gemini Download"
+name: "Gemini"
+manufacturer: "0xCB"
+board_url:
+ - "https://github.com/0xCB-dev/0xCB-Gemini"
+board_image: "0xcb_gemini.jpg"
+date_added: 2024-11-11
+family: raspberrypi
+features:
+  - USB-C
+  - Breadboard-Friendly
```

Comment:
We recently added `Castellated Pads` as one of the filterable features, can you add that to this list please